packages/sdk/src/core/contextInternal.ts

```diff
@@ -114,23 +135,24 @@ export class ContextInternal {
 
 		// check permission for exclusive actors
 		let user: User;
-		if (actor.exclusiveToUser &&
-			(user = this.userSet.get(actor.exclusiveToUser)) &&
-			!(user.grantedPermissions.includes(Permissions.UserInteraction))) {
-			actor.internal.notifyCreated(false,
-				`Permission denied on user ${user.id} (${user.name}). Either this MRE did not ` +
-				"request the UserInteraction permission, or it was denied by the user."
-			);
+		if (actor.exclusiveToUser) {
+			user = this.userSet.get(actor.exclusiveToUser);
```

Comment:
TODO: this should be fixed before merge

Reply:
Is the code getting uncommented as a part of the merge?

---
In reply to: [449561360](https://github.com/microsoft/mixed-reality-extension-sdk/pull/652#discussion_r449561360) [](ancestors = 449561360)

Reply:
Added user.hasGrantedPermissions

---
In reply to: [450333599](https://github.com/microsoft/mixed-reality-extension-sdk/pull/652#discussion_r450333599) [](ancestors = 450333599,449561360)